src/mmw/js/src/modeling/models.js

```diff
@@ -10,6 +10,24 @@ var Backbone = require('../../shim/backbone'),
     turfErase = require('turf-erase'),
     turfIntersect = require('turf-intersect');
 
+var GWLFE = 'gwlfe';
+var TR55_TASK = 'tr55';
+var TR55_PACKAGE = 'tr-55';
+
+var modelPackages = [
+    {
+        name: TR55_PACKAGE,
+        displayName: 'Site Storm Model',
+        description: ''
+    },
+    {
+        name: GWLFE,
+        displayName: 'Watershed Multi-Year Model',
+        description: '',
+        disabled: true
```

Comment:
I thought about doing this, but the backend doesn't need access to this list of allowed packages, and I would have to define the constants again on the frontend.